.github/workflows/nightly-release.yml

```
@@ -414,13 +424,13 @@ jobs:
       - name: Download flashinfer-jit-cache artifact
         uses: actions/download-artifact@v4
         with:
-          name: jit-cache-cu${{ matrix.cuda == '12.9' && '129' || '130' }}-x86_64
+          name: jit-cache-${{ matrix.cuda.label }}-x86_64
           path: dist-jit-cache/
 
       - name: Get Docker image tag
         id: docker-tag
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:3:40: Double quote to prevent globbing and word splitting

(shellcheck)